app/models/guest.rb

```diff
@@ -0,0 +1,4 @@
+class Guest < ApplicationRecord
+  has_many :guest_rooms
+  has_many :rooms, through: :guest_rooms
```

Comment:
With this setup, you can access a guest's room, and then because of the hotel association with room, you can also access a hotel from a room. Taken together, this means you can do:
`guest.rooms.each { |room| room.hotel.name }`

This is helpful for US#1